autopush/main.py

```diff
@@ -101,13 +101,25 @@ def add_external_router_args(parser):
                         type=str, env_var="GCM_APIKEY")
     # Apple Push Notification system (APNs) for iOS
     label = "APNS Router:"
-    parser.add_argument('--apns_sandbox', help="%s Use Dev Sandbox",
+    parser.add_argument('--apns_sandbox', help="%s Use Dev Sandbox" % label,
                         type=bool, default=True, env_var="APNS_SANDBOX")
     parser.add_argument('--apns_cert_file',
                         help="%s Certificate PEM file" % label,
                         type=str, env_var="APNS_CERT_FILE")
-    parser.add_argument('--apns_key_file', help="%s Key PEM file",
+    parser.add_argument('--apns_key_file', help="%s Key PEM file" % label,
                         type=str, env_var="APNS_KEY_FILE")
+    # UDP
+    parser.add_argument(
+        '--udp_timeout',
+        help="UDP: idle timeout before closing socket",
+        type=int, default=0, env_var="UDP_TIMEOUT")
+    parser.add_argument(
+        '--udp_pem',
+        help="UDP: custom TLS PEM file for remote Wake server",
+        type=str, env_var="UDP_PEM")
+    parser.add_argument(
+        '--udp_server', help="UDP: endpoint for UDP wake-up calls",
+        type=str, default='http://example.com', env_var="UDP_SERVER")
```

Comment:
Nit: Inconsistent dropping of arguments. All the existing add_argument calls keep as much on the top line as possible before dropping/indenting.